core/core-api/src/main/resources/plc-coordinate/tr30.json

```
@@ -0,0 +1,17 @@
+{
+  "isConnected": {
+    "type": "Bool",
```

please use camelcase
```suggestion
    "type": "bool",
```